Classes/BEMAnalogClockView.m

```diff
@@ -305,12 +311,23 @@ - (void)handlePan:(UIPanGestureRecognizer *)recognizer {
     self.oldMinutes = self.minutes;
     self.minutes = ((atan2f((translation.x - self.frame.size.height/2), (translation.y - self.frame.size.width/2)) * -(180/M_PI) + 180))/6;
     
-    if (self.oldMinutes > 45 && self.minutes < 15) { // If the user drags the minute hand from 59 to 00, updates the hour on the clock.
-        self.hours++;
+    if (self.touchHourHand == NO) {
+        if (self.oldMinutes > 45 && self.minutes < 15) { // If the user drags the minute hand from 59 to 00, updates the hour on the clock.
+            self.hours++;
+        }
+        else if (self.oldMinutes < 15 && self.minutes > 45) { // If the user drags the minute hand from 00 to 59, updates the hour on the clock.
+            self.hours--;
+        }
     }
-    else if (self.oldMinutes < 15 && self.minutes > 45) { // If the user drags the minute hand from 00 to 59, updates the hour on the clock.
-        self.hours--;
+    else {
+        if ((self.oldMinutes < self.minutes || (self.oldMinutes == 59 && self.minutes == 0)) && self.minutes % 5 == 0) {
```

Comment:
It seems odd to rely on minutes to "directly" operate hours (here and below). I presume this is a sign that this mode requires more deep refactoring. Are you sure this is OK?